### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by YaPanda98**

### CODE REVIEW

File: Worker.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+public class Worker implements ISalary{
+    private int hours;
+    private int days;
+    private int sickLeaves;
+    private int dayOnDemand;
+    private int salary;
+
+
+    // 1 is hourly and 0 for fixed salary. By default, it is fixed
+    private int SALARY_TYPE = 0;
```
Comment (oleksandr-kazimirov) [inline Worker.java:13]: static final for constant

File: Worker.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            salary = 8;
+        }else{
+            System.out.println("Invalid salary type entered! ");
+        }
+
+    }
+
+    public void reportSalary() {
+        double sal = calculateSalary();
+        if(sal < 0)
```
Comment (oleksandr-kazimirov) [inline Worker.java:42]: if(){
}else{
}

**Review by oleksandr-kazimirov — APPROVED** [review]
@YaPanda98 good enough, for now, I left minor comments 👍

**Comment by YaPanda98** [comment]
Good afternoon!
Could you please check my pull request from 21st of July, which is called
if i am not mistaken  "Pointer-abstract" and "Pointer -pet". I`ve made some
changes to that file and i`m looking forward to your feedback☺

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Без
вирусов. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by YaPanda98** [comment]
Best regards, Y.S

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Без
вирусов. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Pull request closed without merging**